xcube_cds/store.py

```diff
@@ -478,7 +478,8 @@ def _create_time_range(t_start: str, t_end: str, t_interval: str):
         dt_start = dateutil.parser.isoparse(t_start)
         dt_end = datetime.datetime.now() if t_end is None \
             else dateutil.parser.isoparse(t_end)
-        period_number, period_unit = CDSDataOpener._parse_time_period(t_interval)
+        period_number, period_unit = \
+            CDSDataOpener._parse_time_period(t_interval)
```

Comment:
Why split this into two lines?

Reply:
I try to keep xcube-cds code style compliant with the [xcube developer guide](https://xcube.readthedocs.io/en/latest/devguide.html#coding-style), which recommends PEP 8 style, which recommends [a maximum line length of 79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

Reply:
See [this reply](https://github.com/dcs4cop/xcube-cds/pull/21#issuecomment-724790253).